cla_backend/apps/reports/sql/MICB1sSLA.sql

```diff
@@ -119,18 +119,22 @@ select
    WHEN source IN ('WEB', 'PHONE') AND operator_first_log_after_cb1__created IS NULL AND now() < callback_window_start THEN FALSE
    WHEN source IN ('WEB', 'PHONE') AND operator_first_log_after_cb1__created IS NULL THEN now() NOT BETWEEN callback_window_start AND callback_window_end
    WHEN source IN ('WEB', 'PHONE') THEN operator_first_log_after_cb1__created NOT BETWEEN callback_window_start AND callback_window_end
-   WHEN operator_first_log_after_cb1__created IS NULL THEN now() > sla_120
-   ELSE operator_first_log_after_cb1__created > sla_120
+   -- User has NOT been contacted and current time is after the SLA1 window for SMS and VOICE MAIL
+   WHEN operator_first_log_after_cb1__created IS NULL THEN now() > "Date_Case_Created" + interval '2 hours'
+   -- User has been contacted and current time is after the SLA1 window for SMS and VOICE MAIL
```

Comment:
Is this comment correct? Is it not "contact time is after the..." rather than the current time?

Reply:
Good spot, updated now